test/common/upstream/upstream_impl_test.cc

```diff
@@ -2231,7 +2397,7 @@ TEST_F(ClusterInfoImplTest, OneofExtensionProtocolOptionsForUnknownFilter) {
         "@type": type.googleapis.com/google.protobuf.Struct
   )EOF";
 
-  EXPECT_THROW_WITH_MESSAGE(makeCluster(yaml), EnvoyException,
+  EXPECT_THROW_WITH_MESSAGE(makeCluster(yaml, false), EnvoyException,
```

Comment:
Ditto, etc.

Reply:
@htuch in my view this also required boosting as field "extension_protocol_options " is not defined in V3 and this particular test case required both to present (typed_extension_protocol_options  & extension_protocol_options )and then test.
see exception output "Only one of typed_extension_protocol_options or extension_protocol_options can be specified"

Reply:
@htuch similar case for few more test cases where those test cases particularly required "extension_protocol_options " field to be present to test functionalities, so boosting is preferred. please suggest.

Reply:
I'd suggest updating the test to use `typed_extension_protocol_options` instead.

Reply:
Actually, I see what's going on. This one is OK.